packages/flutter_tools/lib/src/macos/build_macos.dart

```diff
@@ -41,8 +41,8 @@ Future<void> buildMacOS(FlutterProject flutterProject, BuildInfo buildInfo) asyn
     '-configuration', '$config',
     '-scheme', 'Runner',
     '-derivedDataPath', flutterBuildDir.absolute.path,
-    'OBJROOT=${fs.path.join(flutterBuildDir.absolute.path, 'Build', 'Intermediates.noindex')}',
-    'SYMROOT=${fs.path.join(flutterBuildDir.absolute.path, 'Build', 'Products')}',
+    'OBJROOT=/${fs.path.join(flutterBuildDir.absolute.path, 'Build', 'Intermediates.noindex')}',
+    'SYMROOT=/${fs.path.join(flutterBuildDir.absolute.path, 'Build', 'Products')}',
```

Comment:
Wouldn't the absolute path already start with /?

I was assuming that the issue I'm seeing is due to Generated.xcconfig having
```
SYMROOT=${SOURCE_ROOT}/../build/macos
```

(I'm not clear on why it's in Generated.xcconfig given that it's passed explicitly here, actually.)

Reply:
I think we're overriding that configuration via the xcode build command here. When building from xcode we must be hitting the generated configuration, which is incorrect. I can patch that instead...

Reply:
Fixed